02_getting_started.asciidoc

```diff
@@ -36,33 +36,48 @@ If users decide to outsource some functionality to a third party they usually gi
 
 Finally, those seeking simplicity and convenience, even at the expense of control and security, may choose a custodial Lightning wallet. This is the least technically challenging option, but it _undermines the trust model of cryptocurrency_ and should therefore be considered only as a stepping stone towards more control and self-reliance.
 
-In <<lnwallet-categories>> we see the three broad categories of lightning wallets and the relative degree of control they offer to the user.
+There are many ways wallets can be characterized or categorized. 
+From all features the two most important criteria are LN Node use and fund custody. Correspondingly the two most important questions to ask about a specific wallet are:
+
+- Does this Lightning wallet have a full LN Node or does it use a third-party LN Node?
+- Does this Lightning wallet store its own keys under user control (self-custody) or are the keys held be a third-party custodian?
+
+From these two questions we can derive four possible categories. We can place these four categories into a quadrant as seen in Table <<lnwallet-categories>>. 
+But remember that this is just one way of categorizing Lightning wallets.
 
 [[lnwallet-categories]]
-.Broad Categories of "Lightning Wallets"
+.Lightning Wallets Quadrant
 |===
-| Wallet Type          | LN Node        | Keystore/Custody | Technical Skill
-| Full Node & Wallet   | Full Node      | Non-Custodial    | High
-| Non-Custodial Wallet | 3rd-party node | Non-Custodial    | Medium
-| Custodial Wallet     | 3rd-party node | Custodial        | Low
+|                        | *Full LN Node*      | *3rd-party LN Node*    
+| *Self-Custody*         | Q1: High technical skill | Q2: Medium technical skills 
+| *Custodial*            | Q3: not a common wallet type | Q4: Low technical skills 
 |===
 
+Q3, quadrant 3, where a full LN node is used but the keys are held by a custodian is not common. A user who goes through the effort of running a Ligtning Node will most likely also put in the additional efffort to store his own keys. 
+
 Lightning wallets can be installed on a variety of devices, including laptops, servers, and mobile devices. To run a full Lightning node (one that participates in "gossip" and creates its own map of the network for path finding and routing) you will need to use a server or desktop computer, as mobile devices and laptops are usually not powerful enough in terms of capacity, processing, battery life, and connectivity.
 
+The category "Third-party Ligntning Nodes" can again be subdivided into:
+
+- Lightweight: means that the Lightning Node is operated by a third party and that the wallet obtains the required information through an API that connects wallet and third-party LN node.
```

Comment:
Hmm, I think we're getting close here, but there's still more nuance. One example is the new Phoenix wallet. The user controls the keys, grabs blockchain data from Electrum servers (I believe, making them SPV in a sense), but delegates stuff like opening+managing channels and path finding to a 3rd party server. 

It sits somewhere in the middle, making certain tradeoffs to provide a better user experience in a mobile setting. 

Something on the far opposite end is something like Bluewallet, or other fully custodial wallets. In this case, the user has minimal control and trusts the operator of the hub node to not run away with funds. In the example of the Phoenix wallet, the user still maintains control of their keys.

Reply:
How is this not captured accurately in the table?
Phoenix Wallet | Mobile  | LN=Lightweight | BTC=Electrum | Self-Custody

Let's look at your sentence:
Phoenix wallet. The user controls the keys,  --> Self-custody
grabs blockchain data from Electrum servers (I believe, making them SPV in a sense),  --> Electrum
 but delegates stuff like opening+managing channels and path finding to a 3rd party server --> LN=3rd-party Lightweight

For Phoenix: As far as I can see your sentence matches exactly the table. What is it that I am not seeing?
Same for Blue Wallet. 
Unless I am missing something, it is OK as is. Please give further details to clarify. Thanks.